micro-ui/web/micro-ui-internals/packages/modules/workbench/src/pages/employee/MDMSView.js

```
@@ -30,22 +31,13 @@ const MDMSView = ({ ...props }) => {
     }
   });
 
+  let propsToSendButtons = {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'propsToSendButtons' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)